source/common/protobuf/utility.cc

```diff
@@ -656,8 +657,11 @@ void MessageUtil::unpackTo(const ProtobufWkt::Any& any_message, Protobuf::Messag
                                          any_message_with_fixup.DebugString()));
       }
       Config::VersionConverter::annotateWithOriginalType(*earlier_version_desc, message);
-      MessageUtil::onVersionUpgradeDeprecation(any_full_name);
-      return;
+      auto itr = exclude_v2_protos.find(any_full_name);
+      if (itr == exclude_v2_protos.end()) {
+        MessageUtil::onVersionUpgradeWarn(any_full_name);
```

Comment:
Seems like this is no longer defined(?).

Quoted from the build log https://dev.azure.com/cncf/4684fb3d-0389-4e0b-8251-221942316e06/_apis/build/builds/61997/logs/60.

```
2020-12-21T18:11:46.3595764Z source/common/protobuf/utility.cc:662:22: error: no type named 'onVersionUpgradeWarn' in 'Envoy::MessageUtil'
2020-12-21T18:11:46.3596266Z         MessageUtil::onVersionUpgradeWarn(any_full_name);
2020-12-21T18:11:46.3596575Z         ~~~~~~~~~~~~~^
2020-12-21T18:11:46.3596841Z 1 error generated.
```

Reply:
Ohh. let me check this. thanks @dio

Reply:
@dio strange for me. how come unit test cases are passing then. i run bazel test --test_output=streamed test/common/protobuf:utility_test --cache_test_results=no

Reply:
https://github.com/envoyproxy/envoy/commit/0e42452a48a8e8d10f5a230a94acb7e5fe84e38f#diff-4144a72bc575d857e932f103f6b10bbeb15a938fb8d853533dd8f7da72f0279dR380 :)

Reply:
@dio that's really helpful. thanks a lot !!!